tests/ci/ccache_utils.py

```diff
@@ -57,12 +58,19 @@ def dowload_file_with_progress(url, path):
 
 
 def get_ccache_if_not_exists(
-    path_to_ccache_dir, s3_helper, current_pr_number, temp_path
+    path_to_ccache_dir: str,
+    s3_helper: S3Helper,
+    current_pr_number: int,
+    temp_path: str,
+    release_pr: int,
 ) -> int:
     """returns: number of PR for downloaded PR. -1 if ccache not found"""
     ccache_name = os.path.basename(path_to_ccache_dir)
     cache_found = False
     prs_to_check = [current_pr_number]
+    # Release PR is either 0 or defined
+    if release_pr:
+        prs_to_check.append(release_pr)
```

Comment:
I don't understand this comment - 0 is defined

Reply:
release_pr is set to 0 by default in https://github.com/ClickHouse/ClickHouse/pull/41328/files#diff-b77be45e955311639fd63279b380954c2acd8a0d6c9e090064cf838a4ee9a2baR89. If we are in the backport branch, then [pr-backport](https://github.com/ClickHouse/ClickHouse/blob/c2b02c2ae95d697b63324d9ebe423d903aeb3032/tests/ci/pr_info.py#L185) label should be set, and [release_pr](https://github.com/ClickHouse/ClickHouse/blob/c2b02c2ae95d697b63324d9ebe423d903aeb3032/tests/ci/pr_info.py#L199) attribute is set to a proper release PR. E.g., #40356 for 22.8.

See [here](https://github.com/ClickHouse/ClickHouse/actions/runs/3040217221/jobs/4896014920#step:5:116), it has additional `--cache-from type=registry,ref=clickhouse/keeper-jepsen-test:40356`, the number comes from that attribute.

Reply:
oh, ok - I just mixed it with python meaning of defined - i.e. not None :)